unix/mpconfigport.h

```diff
@@ -81,6 +81,10 @@
 #define MICROPY_PY_BUILTINS_COMPILE (1)
 #define MICROPY_PY_BUILTINS_NOTIMPLEMENTED (1)
 #define MICROPY_PY_BUILTINS_POW3    (1)
+#if !defined(MICROPY_PY_BUILTINS_HELP)
```

Comment:
Please do this idiom the same way it's already done in this file (hint: #ifndef, no indent).